addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -291,6 +326,21 @@ func get_running_event(name: String) -> ESCEvent:
 	return _running_events[name] if name in _running_events else null
 
 
+# Setter for _changing_scene.
+#
+# #### Parameterse
+# - value: boolean value to set _changing_scene to
+func set_changing_scene(value: bool) -> void:
+	escoria.logger.trace("Setting _changing_scene to %s." % value)
+
+	_changing_scene = value
+	
+	# If we're changing scenes, interrupt any (other) running events and purge
+	# all event queues.
+	if value:
```

Comment:
This changes makes the test clearer:
```suggestion
	if _changing_scene:
```
Also possible to rename the `value` variable for `p_is_changing_scene` instead.